build.gradle

```diff
@@ -257,6 +257,45 @@ task buildElasticsearchLocalDistro(dependsOn: unzipDownloadedElasticsearchSource
     }
 }
 
+task buildElasticsearchLogstashBridge(type: Exec) {
+    description "builds logstash-bridge lib module"
+
+    dependsOn buildElasticsearchLocalDistro
+    mustRunAfter buildElasticsearchLocalDistro
```

Comment:
Given that there is already a dependency on `buildElasticsearchLocalDistro` task, this mustRunAfter is redundant I think.

Reply:
Yup, I have removed it.